source/_includes/installation/operating_system.md

```diff
@@ -359,6 +359,8 @@ Minimum recommended assignments:
     8. Select device type: **unix**.
     9. Select name: **org.qemu.guest_agent.0**.
     10. Finally, select **Begin Installation** (upper left corner).
+ 
+    If you use a macvtap network adapter, [allow Zero-configuration/multicast traffic in libvirt](/integrations/zeroconf/#libvirt-virtual-machine-with-macvtap-adapter).
```

Comment:
These instructions document using a bridge, not macvtap.

If you want to divert from our instructions, you'll end up with an unsupported installation type. That is fine, but that also means you may encounter issues (like the one in this PR), but you'll have more issues (e.g., missing DHCP snooping).

Therefore, we should not document this, as users might thing this is an acceptable/supported solution (which it is not).

Nevertheless, thanks for being willing to contribute 👍 

../Frenck